src/main/java/seedu/address/logic/parser/AssignCommandParser.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DRIVER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EVENT_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK;
+
+import java.time.format.DateTimeParseException;
+
+import seedu.address.logic.commands.AssignCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.EventTime;
+
+/**
+ * Parses input arguments and creates a new EditCommand object
+ */
+public class AssignCommandParser implements Parser<AssignCommand> {
+
+
+    private static ParseException getWrongFormatException() {
+        return new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AssignCommand.MESSAGE_USAGE));
+    }
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the EditCommand
+     * and returns an EditCommand object for execution.
+     *
+     * @return the parsed command
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AssignCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_DRIVER, PREFIX_TASK, PREFIX_EVENT_TIME);
+
+
+        // check if the force flag is correctly spelled
+        boolean isForce = argMultimap.getPreamble().equals("force");
+        boolean isEmpty = argMultimap.getPreamble().isEmpty();
+
+        if (isEmpty == isForce) {
+            throw getWrongFormatException();
+        }
```

Comment:
Would be better to create a ParseUtil.parseForce, trim and check the string in that method. Because "force " or "    force" should work too.

Reply:
`getPreamble()` already trims the prevailing and trailing spaces.